test/cms/repo_test.exs

```diff
@@ -97,6 +97,16 @@ defmodule CMS.RepoTest do
                "cms.repo" <> String.replace(path, "/", "|") <> "?bam=bop&baz=qux"
     end
 
+    test "generates the correct key for a map" do
+      path = "/foo/bar"
+      params = %{"biz" => "bang", "data" => %{"some" => "map"}}
+
+      assert Repo.generate(nil, nil, [path, params]) ==
+               "cms.repo" <> String.replace(path, "/", "|") <> "?biz=bang&data=?some=map"
```

Comment:
💥 This won't be parseable by the CMS. `?biz=bang&data=?some=map` will just take `?some=map` as a string when you get the params.

Reply:
This is the structure that is being passed in: `?schedule_finder[direction_id]=1&schedule_finder[origin]=3939`

Reply:
Also, I can confirm that we pass those params on to the CMS and they are ignored.

Reply:
> Also, I can confirm that we pass those params on to the CMS and they are ignored.

There is a whitelist filter for CMS params somewhere in the dotcom code. Not sure how it interacts with this stuff but you may want to track it down.

Reply:
If the CMS ignores the params, then I am just going to reject the map

Reply:
There might be cases where we don't ignore the params. That's why I'm thinking just make the params to string function that the CMS code is using public and then use that to make the key. That way they're guaranteed to be the same.